src/platforms/windows/daemon/windowssplittunnel.cpp

```diff
@@ -673,3 +691,25 @@ bool WindowsSplitTunnel::detectConflict() {
   CloseServiceHandle(servicehandle);
   return err == ERROR_SERVICE_DOES_NOT_EXIST;
 }
+
+bool WindowsSplitTunnel::isRunning() { return getState() == STATE_RUNNING; }
+
+QString WindowsSplitTunnel::stateString() {
```

Comment:
Adding this for better logs, in santiagos logs you can see state is `18446744073709551615` which is -1 as uint64 😅